syncer/syncer.go

```diff
@@ -267,6 +273,20 @@ func (s *Syncer) Start() {
 				}
 			}
 		})
+		if !s.cfg.Standalone {
+			s.eg.Go(func() error {
```

Comment:
i think it will be better if background jobs were spawned in the same place. in the syncAtx method we do synchronous requests and then spawn background job to check atxs from peers, maybe spawn background job for malfeasance sync there as well?

Reply:
done